docs/src/interface.md

```diff
@@ -0,0 +1,61 @@
+# Interface
+
+## Edge
+
+```@docs
+AbstractEdge
+src
+dst
+weight
+Base.reverse
+```
+
+## Graph (required)
+
+```@docs
+AbstractGraph
+is_directed
+vertices
+out_edges
+in_edges
+```
+
+## Graph (optional)
+
+```@docs
+nv
+ne
```

Comment:
I really wish these horrible names would be removed or at least deprecated...

Reply:
That would at least fit with `ndigits`, `nfields`, `ndims`.

(Personally I've come to loath all these abbreviations -- they are super nice if you bang away on experiments in the REPL. But write code with them and come back after a year and try to read it, and it looks like gobbledygook. Esp. if you have to context switch between multiple languages and libraries... So I am now tending into the more "radical" direction of preferring names like `num_vertices` or even `number_of_vertices`. But I realize I am probably in the minority on this, which is fine, but I figure I should at least mention it once before staying silent forever after ;-). Anyhow, thank you for working on this ).

Reply:
Yeah, `nvertices` and `nedges` seems decent.